crates/astria-sequencer-relayer/src/relayer/write/mod.rs

```diff
@@ -264,21 +290,20 @@ impl BlobSubmitter {
 /// Submits new blobs Celestia.
 ///
 /// # Panics
-/// Panics if `blobs` or `sequencer_heights` are empty. This function
-/// should only be called if there is something to submit.
+/// Panics if `blocks` is empty. This function should only be called if there is something to
+/// submit.
 #[instrument(
     skip_all,
-    fields(
-        num_blobs = blobs.len(),
-        sequencer_heights = %ReportSequencerHeights(&sequencer_heights),
-))]
+    fields(blocks = %telemetry::display::json(&blocks.infos))
```

Comment:
All this information gets injected into the span. My reasoning is that I wanted to have it in every single event in here. Maybe that's excessive and should just be logged in the first event?

Reply:
Could we tie a uuid or something like that to the blocks? Then we can trace with a UUID and have blocks.info in the first debug log.

Reply:
This should technically be handled by our observability platform. Inject UUIDs manually duplicates that work.

I have removed the field from the span, emitting all this data in an event at the start of the submission tasks instead.